superset-frontend/src/components/Menu/Menu.tsx

```diff
@@ -90,6 +90,9 @@ const StyledHeader = styled.header`
     flex-direction: column;
     justify-content: center;
   }
+  .main-nav .ant-menu-submenu-title > svg {
```

Comment:
do you need .main-nav for specificity or will it work without it?

Reply:
I tested to make sure, it does need this specificity.